### PR TITLE
[Bug] Switch pip camera button accessibility

### PR DESCRIPTION
**Pull request by ShaunaSong**
## Purpose

* Announce the front/ back camera when user tap on the pip camera image button

## Does this introduce a breaking change?

```
[ ] Yes
[x] No
```

## Pull Request Type
What kind of change does this Pull Request introduce?

```
[x] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:
```

## How to Test

- Turn on the Talkback
- Navigate to call setup page.
- Navigate to Join call and activate it.
- Turn Video ON
- Navigate to PIP of local video.
- Get focus on 'Switch camera button'  and activate it.
- Observe the issue.

## What to Check
Verify that the following are valid
* Talkback announce the front/ back camera

### CODE REVIEW

**Review by dhiraj — CHANGES_REQUESTED** [review]

File: azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AccessibilityAnnouncementManager.kt [hunk 1]
```diff
@@ -102,6 +103,19 @@ internal class MeetingJoinedHook : AccessibilityHook() {
...
 
+internal class PipCameraStatusHook : AccessibilityHook() {
+    override fun shouldTrigger(lastState: ReduxState, newState: ReduxState): Boolean =
+        (lastState.localParticipantState.cameraState.device != newState.localParticipantState.cameraState.device)
+
+    override fun message(lastState: ReduxState, newState: ReduxState, context: Context): String {
+        return when (newState.localParticipantState.cameraState.device) {
+            CameraDeviceSelectionStatus.FRONT -> context.getString(R.string.azure_communication_ui_calling_switch_camera_button_back)
+            CameraDeviceSelectionStatus.BACK -> context.getString(R.string.azure_communication_ui_calling_switch_camera_button_front)
+            else -> ""
```
[the remark this answers is not in the document]
Comment (ShaunaSong) [reply azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AccessibilityAnnouncementManager.kt:114]: Current logic: 
if it is FRONT, and the talkback will announce "Switch to Back Camera button, double click to activate"
If it is BACK,  and will announce "Switch to Front Camera button, double click to activate"

I think it is also make sense.

**Review by lpeixin — COMMENTED** [review]
Tested

File: azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AccessibilityAnnouncementManager.kt (hunk 1 above)
[the remark this answers is not in the document]
Comment (dhiraj) [reply azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AccessibilityAnnouncementManager.kt:114]: No, I meant when you're already on focus on the camera button, and you double tap then the announcement sounds like it is opposite. For example, say you're focused on the camera button, and the currently selected camera is front, when you double tap the button, it switches to the back camera, but the announcement is "Switch to front camera".

[the remark this answers is not in the document]
Comment (ShaunaSong) [reply azure-communication-ui/azure-communication-ui/src/calling/java/com/azure/android/communication/ui/calling/presentation/manager/AccessibilityAnnouncementManager.kt]: Get it. Make sense. Also check with Mic on/off logic, will follow the same logic. 

Thanks for helping.

**Review by dhiraj — APPROVED** [review]
Tested, looks good to me 👍

**Review by lpeixin — APPROVED** [review]

**Pull request merged**